src/main/java/platform/codingnomads/co/springweb/springrestcontrollers/simpledemo/controller/HelloWorldController.java

```diff
@@ -16,6 +19,18 @@ public String hello() {
     public String greeting(@PathVariable(name = "name") String name) {
         return "Hello " + name + "!";
     }
+
+    @RequestMapping(path = "/hello/list", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public String helloList() {
+        List<Integer> integerList = List.of(1, 2, 3, 4);
+        return integerList.toString();
+    }
```

Comment:
This implementation does not produce JSON but is just due to the `.toString()` implementation of the `List` interface. 

The correct/safe way would be to use `List` as the return type:
```suggestion
    @RequestMapping(path = "/hello/list", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
    public List helloList() {
        List<Integer> integerList = List.of(1, 2, 3, 4);
        return integerList;
    }
```